config.json

```diff
@@ -395,12 +394,17 @@
           "functions",
           "loops"
         ]
+      },
+      {
+        "slug": "all-your-base",
+        "name": "All Your Base",
+        "uuid": "02d07b3b-86a2-404f-9d95-170602315a35",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 5
       }
     ]
   },
-  "foregone": [
```

Comment:
We should be keeping `foregone`.

Multithreading exercises like `"parallel-letter-frequency"` and `"bank-account"` are not applicable for this track.

Reply:
I think that was remove by configlet when I ran `configlet fmt`. I think it makes sense to forgo those exercises, so I'll add them in.